server/services/v1/auth_provider.go

```diff
@@ -155,18 +168,18 @@ func (a *Auth0) UpdateApplication(ctx context.Context, req *api.UpdateApplicatio
 		return nil, err
 	}
 
-	metadata := client.GetClientMetadata()
-	metadata[updatedBy] = currentSub
-	metadata[updatedAt] = time.Now().Format(time.RFC3339)
+	m := client.GetClientMetadata()
+	m[updatedBy] = currentSub
+	m[updatedAt] = time.Now().Format(time.RFC3339)
 	appToUpdate := &management.Client{
 		Name:           &req.Name,
 		Description:    &req.Description,
-		ClientMetadata: metadata,
+		ClientMetadata: m,
 	}
 
 	err = a.Management.Client.Update(req.GetId(), appToUpdate)
```

Comment:
Not updating cache in after upate?

Reply:
Means token for old credentials will live there forever?

Reply:
no we delete them prior to rotation.

Reply:
got it. but it not obvious.